packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -91,7 +91,7 @@ export class Dropdown extends BaseComponent<IDropdownProps, IDropdownState> {
           aria-expanded={ isOpen ? 'true' : 'false' }
           role='combobox'
           aria-live={ disabled || isOpen ? 'off' : 'assertive' }
-          aria-label={ label }
+          aria-label={ ariaLabel }
```

Comment:
I think we should fall back to `label` here if `ariaLabel` is not set. Both for future usage and for existing users who would be broken otherwise